src/current/v24.3/ldap-authentication.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Configure LDAP Authentication
+summary: Learn how to configure CockroachDB to use LDAP for user authentication with directory services like Active Directory.
+toc: true
+---
+
+CockroachDB supports authentication and authorization using LDAP (Lightweight Directory Access Protocol) with services such as Active Directory. This allows you to integrate your existing directory service with CockroachDB for centralized user management and access control.
```

Comment:
```suggestion
CockroachDB supports authentication and authorization using identities stored in LDAP (Lightweight Directory Access Protocol) with services such as Azure Active Directory. This allows you to integrate your cluster with your organization's existing identity infrastructure, allowing for centralized user management and access control.
```

Reply:
```suggestion
CockroachDB supports authentication and authorization using LDAP-compatible directory services, such as Active Directory and Microsoft Entra ID. This allows you to integrate your cluster with your organization's existing identify infrastructure, enabling centralized user management and access control.
```

Further edit for consideration. I figure we should integrate the [new name for Azure AD](https://learn.microsoft.com/en-us/entra/fundamentals/new-name), while also making reference to the more well-known Active Directory (which still exists / is still true; per that link "Active Directory" remains correct terminology as a shorthand for "Windows Server Active Directory".

Reply:
Cool, I didn't know there was a new name. My instinct to add "Azure" was for SEO and that I'd be very surprised if our ICP tend to run internal non-Azure Active Directory instances that they connect to their clusters. I guess I could be wrong.